src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -230,6 +230,33 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
         }
 
+        /// <summary>
+        /// Gets an &lt;input&gt; elements "type" attribute value for the given <paramref name="modelExplorer"/>.
+        /// </summary>
+        /// <param name="modelExplorer">The <see cref="ModelExplorer"/> to inspect.</param>
+        /// <param name="inputTypeHint">When this method returns, contains a type hint associated with the given
+        /// <paramref name="modelExplorer"/>.</param>
+        /// <returns>An &lt;input&gt; elements "type" attribute value.</returns>
+        /// <remarks><paramref name="inputTypeHint"/> values are based off of the <paramref name="modelExplorer"/>'s
+        /// <see cref="ModelExplorer.Metadata"/>. Input type hints are typically used to format an
+        /// &lt;input&gt; elements "value" attribute of a generated text box or used to determine if a text box should
+        /// be multi-line.</remarks>
```

Comment:
Remove the `<remarks/>`. What you're saying here is misleading and not complete enough to be useful.